README.md

```diff
@@ -20,6 +20,21 @@ var gdaxClient = new GDAXSharp.GDAXClient(authenticator);
 
 //use one of the services 
 var allAccounts = await gdaxClient.AccountsService.GetAllAccountsAsync();
+
+//use the websocket feed
```

Comment:
i'd argue this deserves its own box 😄 one for using the services and one for the websocket feed